roborazzi-idea-plugin/src/main/kotlin/com/github/takahirom/roborazzi/idea/preview/PreviewViewModel.kt

```diff
@@ -256,7 +251,18 @@ class PreviewViewModel {
   }
 
   fun onShouldSeeIndexHandled() {
-    shouldSeeIndex.value = -1
+    shouldSeeImageIndex.value = -1
+  }
+
+  fun onTaskExecuted(project: Project) {
+    _dropDownUiState.update { currentUiState ->
+      currentUiState.copy(flag = ActionToolbarUiState.Flag.IDLE)
+    }
+    refreshImageList(project)
+    // For updating the task list after syncing the gradle task
+    coroutineScope.launch {
+      fetchTasks(project)
```

Comment:
I guess we can skip executing this in a coroutine

Reply:
Thanks
I addressed some performance issues in https://github.com/takahirom/roborazzi/pull/439/files